app/commands/assets/soft_delete_relations.rb

```diff
@@ -0,0 +1,18 @@
+module Assets
+  class SoftDeleteRelations
+    attr_reader :asset
+
+    def initialize(asset:)
```

Comment:
Keyword arguments are great when passing in options or needing to manage multiple arguments without having to worry about order. 

However, I'd prefer just to have a simple argument here, since we are just executing the command on the passed in object.